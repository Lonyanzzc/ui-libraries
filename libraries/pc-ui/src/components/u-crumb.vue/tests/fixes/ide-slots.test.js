import {
  describe,
  it,
  vi as jest,
  expect,
} from 'vitest';
  import Vue from 'vue';
import { UCrumb } from '../../index.js';
import { mount } from '@vue/test-utils';

describe('Crumb IDE slots test', () => {
    it('IDE 中空值渲染', () => {
        Vue.prototype.$env.VUE_APP_DESIGNER = true;
        const wrapper = mount(UCrumb);
        expect(wrapper.html()).toMatchSnapshot();
        Vue.prototype.$env.VUE_APP_DESIGNER = false;
    });
});
