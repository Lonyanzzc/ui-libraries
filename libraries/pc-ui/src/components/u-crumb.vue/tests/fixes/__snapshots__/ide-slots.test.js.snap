// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Crumb IDE slots test > IDE 中空值渲染 1`] = `
<nav class="root">
  <div class="empty" s-empty="true"></div>
</nav>
`;
